src/oasis-nexus/api.ts

```diff
@@ -73,12 +73,14 @@ export const isAccountEmpty = (account: RuntimeAccount) => {
 export const isAccountNonEmpty = (account: RuntimeAccount) => !isAccountEmpty(account)
 
 export const groupAccountTokenBalances = (account: Omit<RuntimeAccount, 'tokenBalances'>): RuntimeAccount => {
-  const tokenBalances: Record<generated.EvmTokenType, generated.RuntimeEvmBalance[]> = {
-    ERC20: [],
-    ERC721: [],
-  }
+  const tokenBalances: Partial<Record<generated.EvmTokenType, generated.RuntimeEvmBalance[]>> = {}
   account.evm_balances.forEach(balance => {
-    if (balance.token_type) tokenBalances[balance.token_type].push(balance)
+    if (balance.token_type) {
+      if (!tokenBalances[balance.token_type]) {
+        tokenBalances[balance.token_type] = []
+      }
```

Comment:
or `tokenBalances[balance.token_type] ??= []`

Reply:
Nice, I didn't know this works already.